DESCRIPTION

```diff
@@ -4,7 +4,9 @@ Title: Check Package using Various Versions of Dependencies
 Version: 0.0.0.9000
 Date: 2023-02-14
 Authors@R:
-    person("Pawel Rucki", , , "pawel.rucki@roche.com", role = c("aut", "cre"))
+    person("Pawel Rucki", , , "pawel.rucki@roche.com", role = c("aut", "cre")),
+    person("Andre Verissimo", , , "andre.verissimo@roche.com", role = "aut"),
```

Comment:
@averissimo I have added you to the authors to recognise your contributions so far 🥇